core/src/main/java/com/predic8/membrane/core/interceptor/oauth2/processors/TokenEndpointProcessor.java

```diff
@@ -31,8 +36,18 @@ public boolean isResponsible(Exchange exc) {
     }
 
     @Override
-    public Outcome process(Exchange exc) throws Exception {
-        exc.setResponse(new TokenFlowDecider(authServer,exc).getFlow().validateRequest());
+    public Outcome process(Exchange exc) {
+        try {
+            exc.setResponse(new TokenFlowDecider(authServer,exc).getFlow().validateRequest());
+        } catch (Exception e) {
+            log.error(e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.

Reply:
I'll not comment this pattern further down.